### PR TITLE
(SIMP-1359) Update README.md from puppet strings

### PR DESCRIPTION
**Pull request by silug**
This updates README.md to include puppet strings documentation.

SIMP-1359 #close

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,192 @@
 [![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html) [![Build Status](https://travis-ci.org/simp/pupmod-simp-logrotate.svg)](https://travis-ci.org/simp/pupmod-simp-logrotate) [![SIMP compatibility](https://img.shields.io/badge/SIMP%20compatibility-4.2.*%2F5.1.*-orange.svg)](https://img.shields.io/badge/SIMP%20compatibility-4.2.*%2F5.1.*-orange.svg)
```
Comment (nick-markowski) [inline README.md]: Should update the version compatibility

Comment (nick-markowski) [inline README.md]: Should update the compatibility versions

Comment (silug) [reply README.md]: I agree, but I wasn't sure what should go there.  I also wasn't sure if that should be part of this PR or if it should be submitted separately.

Comment (trevor-vaughan) [reply README.md]: @silug Just remove the compatibility badges for now.

Comment (trevor-vaughan) [reply README.md]: @silug I took care of it.

**Pull request merged**